dhis-2/dhis-services/dhis-service-tracker/src/main/java/org/hisp/dhis/tracker/export/trackedentity/HibernateTrackedEntityStore.java

```diff
@@ -728,16 +690,17 @@ private String getFromSubQueryEnrollmentConditions(
     if (params.hasFilterForEvents()) {
       sql.append("inner join (");
       addEventFilter(sql, sqlParameters, params);
-      sql.append(") EV on EV.enrollmentid = en.enrollmentid ");
+      sql.append(") ev on ev.enrollmentid = en.enrollmentid ");
     }
 
-    sql.append("where en.trackedentityid = TE.trackedentityid ")
-        .append("and en.programid = :enrolledProgramId ");
-    sqlParameters.addValue("enrolledProgramId", params.getEnrolledInTrackerProgram().getId());
+    sql.append("where en.trackedentityid = te.trackedentityid ")
+        .append("and en.programid = :enrolledInTrackerProgram ");
+    sqlParameters.addValue(
+        "enrolledInTrackerProgram", params.getEnrolledInTrackerProgram().getId());
 
     if (params.hasEnrollmentStatus()) {
       sql.append("and en.status = :enrollmentStatus ");
-      sqlParameters.addValue("enrollmentStatus", params.getEnrollmentStatus());
+      sqlParameters.addValue("enrollmentStatus", params.getEnrollmentStatus().name());
     }
 
     if (params.hasFollowUp()) {
```

Comment:
It seems that this is throwing a 500.
Not sure what it is, but it seems that `IS` syntax and passing a `Boolean` object to the `sqlParameters` don't work fine together.